### PR TITLE
README: a more descriptive readme

### PR DESCRIPTION
**Pull request by auag92**
Modifying the readme to use more simple (jargon-free) and descriptive language, while all high lighting all important aspects of PyMKS functionalities and capabilities (applications).

### CODE REVIEW

**Comment by auag92** [comment]
This is still a draft PR. Will notify when it is ready for review.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ alt="Travis CI">
 </a>
 
 
+PyMKS is an open source, pythonic implementation of the methodologies
```
Comment (wd15) [inline README.md:31]: My rewrite of that:

PyMKS is an open source, Pythonic implementation of the techniques developed under the aegis of the Materials Knowledge System (MKS) [refs]. The MKS is a methodology for building process-structure-property linkages for materials science applications. PyMKS provides efficient tools for obtaining a digital, uniform grid representation of a materials internal structure in terms of its local states, and computing hierarchical descriptors of the structure that can be used to used to build efficient machine learning based mappings to the relevant response space.

**Comment by auag92** [comment]
> I like it. I rewrote it a little bit. Let's use it. Replace the first para with it.
> 
> Just add the links for the references via the doi link, don't bother listing them in the readme.
> 
> @beyucel: do you want to take a hack at it after @auag92 changes it?

Thanks Daniel. I will give the whole thing another go tomorrow and then it should be ready for review.

File: README.md (hunk 1 above)
Comment (beyucel) [reply README.md:31]: I think that looks good! Once @auag92 finalizes his draft, I can take a look at it!

**Comment by wd15** [comment]
@auag92 @beyucel, this is 14 days old now. Let's finish it and the I can be in the next release with conda, nix and API updates.

**Comment by auag92** [comment]
@beyucel ready for your review. Modified the introduction - the rest of the structure looks okay as is.

File: README.md [hunk 2]
```diff
@@ -29,27 +29,35 @@ alt="Travis CI">
...
+
+
+PyMKS consists of tools to compute 2-point statistics, tools for both homogenization 
+and localization linkages, and tools for discretizing the microstructure. In addition, 
+PyMKS has modules for generating data sets using conventional numerical simulations 
+and a module for custom visualization of microstructures. 
+
+To learn about PyMKS start with the [PyMKS examples](./index.ipynb), 
+especially the [introductory example](notebooks/intro.ipynb). 
+To learnmore about the methods consult the
```
Comment (wd15) [inline README.md:56]: learnmore -> learn more

Comment (wd15) [reply README.md:56]: Fix this and it LGTM

Comment (beyucel) [reply README.md:56]: also line 46 environemnt -> environment

**Comment by beyucel** [comment]
@auag92 thank you for the beautiful description! I think the question is who are we targetting with this ReadME file? If the answer is 1) General Material Science audience who are interested in ML tools for P-S-P linkages (by utilizing feature extraction etc), then I think this update and current ReadMe files are too MKS oriented ( people who have not familiar with MKS might find it a little bit complicated. If we are targetting 2) Material scientists who are familiar with MKS and want to discover MKS tools then this is good ( I would have slight additions).
What do you think @auag92 @wd15 ?

**Comment by wd15** [comment]
> What do you think @auag92 @wd15 ?

We are targeting 1) for sure. Do you want to have a hack at it to make it more general?

**Comment by wd15** [comment]
Shall we go with this for now since it's better than what we currently have? If anyone wants to fix it further then they can make another PR.

**Comment by beyucel** [comment]
> Shall we go with this for now since it's better than what we currently have? If anyone wants to fix it further then they can make another PR.

That makes sense! I will do a separate PR

**Comment by auag92** [comment]
> @auag92 thank you for the beautiful description! I think the question is who are we targetting with this ReadME file? If the answer is 1) General Material Science audience who are interested in ML tools for P-S-P linkages (by utilizing feature extraction etc), then I think this update and current ReadMe files are too MKS oriented ( people who have not familiar with MKS might find it a little bit complicated. If we are targetting 2) Material scientists who are familiar with MKS and want to discover MKS tools then this is good ( I would have slight additions).
> What do you think @auag92 @wd15 ?

Hey @beyucel, I was indeed attempting to generalize in the second para, but as you correctly pointed - the use of MKS jargon without any preface is definitely confusing. Let's try to make this more understandable in a new PR.

File: README.md [hunk 3]
```diff
@@ -28,20 +28,36 @@ alt="Travis CI">
...
+and model construction step (including hyper parameter optimization). PyMKS, written in a functional 
+programming style and supporting distributed computation (multi-core, multi-threaded, cluster), provides 
+modular functionalities to address each of these data transformation steps, while maximally leveraging 
+the capabilities of the underlying computing environment.
+
+
+PyMKS consists of tools to compute 2-point statistics, tools for both homogenization 
+and localization linkages, and tools for discretizing the microstructure. In addition, 
+PyMKS has modules for generating data sets using conventional numerical simulations 
+and a module for custom visualization of microstructures.
```
[the remark this answers is not in the document]
Comment (auag92) [reply README.md]: @wd15, I have pushed in a new commit with modifications based on your comments.

**Comment by auag92** [comment]
@wd15 I have made edits as per your review. Should I squash the commits to make the branch ready for merge?

**Comment by wd15** [comment]
> @wd15 I have made edits as per your review. Should I squash the commits to make the branch ready for merge?

Don't worry, I think it's ok as is.

**Pull request merged**